metricbeat/module/kubernetes/_meta/config.yml

```diff
@@ -1,6 +1,23 @@
+# Node metrics, from kubelet:
 - module: kubernetes
-  metricsets: ["node","container","volume","pod","system"]
   enabled: false
+  metricsets:
+    - node
+    - system
+    - pod
+    - container
+    - volume
   period: 10s
   hosts: ["localhost:10255"]
 
+# State metrics from kube-state-metrics service:
+- module: kubernetes
+  enabled: false
+  metricsets:
+    - state_node
+    - state_deployment
+    - state_replicaset
+    - state_pod
+    - state_container
+  period: 10s
+  hosts: ["kube-state-metrics:8080"]
```

Comment:
Yes, normally it's that: https://github.com/kubernetes/kube-state-metrics/blob/master/kubernetes/kube-state-metrics-service.yaml#L4